### PR TITLE
toesocks

### PR DESCRIPTION
**Pull request by LariAnjelo**
#### Summary

Adds two new socks, toe socks and split-toe socks.

Category "Itens"

#### Purpose of change

This is a pretty simple addition to the already gigantic list of items to the game. It adds a new unique sock: The toe sock, alongside the split-toe sock. Just for flavor, since they are literally just socks with pockets for your toes (and for me to wear geta with a proper pair of socks). I decided not to add it as an variant to the standard pair of socks, because this would mean that the professions in-game would have a chance of spawning with toe socks, something I'd rather avoid for now to avoid any further issues.

#### Describe the solution

Not doing it.

#### Describe alternatives you've considered

Adding it as a simple variant for the sock, however, this would require me to change the entire list of professions we have so a cop wouldn't start with a toe sock. something I am not used to, instead I'd rather deal with the loot groups and item spawns, because I am more used to them. Please, I hope y'all understand.

#### Testing

Everything should work just fine.

#### Additional context

It's me, Cosmic! The one behind the chewing gum flavors and other monstrosities. Consider this as my comeback: But I'll stick to simpler, smaller things for now own. Fluff is the new black!

### CODE REVIEW

File: data/json/items/armor/boots.json [hunk 1]
```diff
@@ -2703,6 +2703,20 @@
     "flags": [ "VARSIZE", "SKINTIGHT" ],
     "armor": [ { "coverage": 100, "covers": [ "foot_l", "foot_r" ] } ]
   },
+  {
+    "id": "toesocks",
+    "type": "ARMOR",
+    "name": { "str": "pair of toe socks", "str_pl": "pairs of toe socks" },
+    "description": "A pair of comfortable socks that separate the toes: They have separate toe sleeves that work with the natural shape of your foot and your foot's anatomy",
```
Comment (DoctorBoomstick) [inline data/json/items/armor/boots.json:2710]:
```suggestion
    "description": "A pair of comfortable socks that separate the toes: They have separate toe sleeves that accommodate one's natural foot shape and anatomy.",
```

File: data/json/items/armor/boots.json [hunk 2]
```diff
@@ -2703,6 +2703,20 @@
...
+    "copy-from": "socks"
+  },
+  {
+    "id": "splitsocks",
+    "type": "ARMOR",
+    "name": { "str": "pair of split toe socks", "str_pl": "pairs of split toe socks" },
```
Comment (DoctorBoomstick) [inline data/json/items/armor/boots.json:2716]: Any reason in particular why these aren’t just variants of the normal toe socks? There’s no difference in statistics, as far as I can see?

Comment (DoctorBoomstick) [reply data/json/items/armor/boots.json:2716]: I just realised you put this on draft, so I’m wagering you’re planning to add some form of difference, whatever that might look like. If not, though, the first comment applies, and if you are, apologies for being hasty with the review.

**Review by DoctorBoomstick — CHANGES_REQUESTED** [review]

File: data/json/items/armor/boots.json (hunk 2 above)
Comment (LariAnjelo) [reply data/json/items/armor/boots.json:2716]: Greetings Doctor, it is an honor to meet you once again. Now that you mentioned it, I don't see a reason why the split-toe socks are not variants of the same item. I'll fix it right now.

**Comment by anothersimulacrum** [comment]
> I decided not to add it as an variant to the standard pair of socks, because this would mean that the professions in-game would have a chance of spawning with toe socks, something I'd rather avoid for now to avoid any further issues.

It's okay if it does show up - variants have weights, you can simply make them a low weight version of normal socks, and I think that's what you should do.

**Comment by anothersimulacrum** [comment]
Okay. Then set them to a weight of zero, and add entries for the item with the toesock variant(s) to itemgroups. Either way, there is no reason for them not to be a variant.

**Comment by LariAnjelo** [comment]
> Okay. Then set them to a weight of zero, and add entries for the item with the toesock variant(s) to itemgroups. Either way, there is no reason for them not to be a variant.

I'm closing this PR. Too much of a work to such simple stuff. Thanks for demotivating.

**Pull request closed without merging**